### PR TITLE
Added spiderbomb.js so everyone can enjoy it

### PR DESCRIPTION
**Pull request by josephcarmello**
I basically copied the Dibsy Slam (slam.js) file and swapped the command call and the array of images.

### CODE REVIEW

File: src/scripts/spiderbomb.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "https://media.giphy.com/media/OM8JhWiICevDy/giphy.gif",
+    "https://media.giphy.com/media/ERsmR91gnL5Sw/giphy.gif"
+];
+
+for (var i = 0; i < 130; i++) {
+    images.push("https://ak-hdl.buzzfed.com/static/2014-11/22/5/enhanced/webdr01/anigif_enhanced-19058-1416653074-7.gif");
+}
+
+module.exports = function(robot) {
+    return robot.hear(/dibsy (spiderbomb)/i, function(msg) {
```
Comment (andromedado) [inline src/scripts/spiderbomb.js]: The more standard approach is `robot.respond(/.*spiderbomb/i, function (msg) {`

The style `robot.hear` means that even if, heaven forbid, we were the change dibsy's name, this command will still only work with "dibsy"

Comment (cmckendry) [inline src/scripts/spiderbomb.js]: Fun fact: there was actually a week where Dibsy was named "Dibby"

Comment (josephcarmello) [inline src/scripts/spiderbomb.js]: @andromedado Awesome! I'll make the change.

Comment (jballant) [inline src/scripts/spiderbomb.js]: I thought it was a few weeks until people decided the "s" needed to be in the name.

**Comment by melieru** [comment]
NOBODY WILL ENJOY THIS. 

DO NOT MERGE. DO NOT MERGE.

**Comment by josephcarmello** [comment]
ignore @mary1stdibs .... @andromedado everyone will enjoy this.

**Pull request merged**